src/components/portal/common/ProductImage.js

```diff
@@ -23,7 +23,7 @@ class ProductImage extends Component {
             style={{
               opacity: loading ? 0.5 : 1,
               objectFit: 'cover',
-              height: '150px'
+              width: '150px'
```

Comment:
If you set both a height and width, it will distort the aspect ratio.
objectFit: cover crops the image to a square

Reply:
Ok, thank you.